specs/core/0_beacon-chain.md

```diff
@@ -171,7 +158,6 @@ The following values are (non-configurable) constants used throughout the specif
 | Name | Value |
 | - | - |
 | `FAR_FUTURE_EPOCH` | `Epoch(2**64 - 1)` |
-| `ZERO_HASH` | `Hash(b'\x00' * 32)` |
```

Comment:
I really link the removal of this, it's much cleaner and also serves as a reminder to readers that SSZ defaults to `0b0`.

Reply:
I think I want to do the same thing with slots and epochs, i.e. nuke `GENESIS_SLOT` in favour of `Slot()` (and same for epochs).

Reply:
I think it reasonable to keep `GENESIS_SLOT` configurable. That's why I didn't put it in the "constants" section.